### ISSUE TITLE
dotnet-coverage collect how to specify dotnet test runsettings?

### ISSUE THREAD

**Issue by butterworld** [microsoft/vstest#4185]
## Description

Hello, I can't seem to figure out how to pass runtime settings when I'm trying to use dotnet-coverage collect with dotnet test.  I tried the following (the parameter that's I'm trying to pass in is NUnit.DisplayName=FullName):
`
dotnet-coverage collect dotnet test ./UnitTest/UnitTest.csproj --framework net48 -c Release /p:Platform=x64 --blame --no-build --no-restore --verbosity normal -r Results --logger "console;verbosity=normal" --logger "trx;LogFileName=UnitTest_results.trx" --filter "TestCategory!=Category1 & TestCategory!=Category2" -- NUnit.DisplayName=FullName --output-format cobertura --output Results/UnitTest.cobertura.xml
`
which complains gives me msbuild msb1008 error -- NUnit.DisplayName=FullName being an invalid switch.  

I also tried to create a .runsettings file and run it as follows:
`
dotnet-coverage collect dotnet test ./UnitTest/UnitTest.csproj --framework net48 -c Release /p:Platform=x64 --blame --no-build --no-restore --verbosity normal -r Results --logger "console;verbosity=normal" --settings ./test.runsettings --logger "trx;LogFileName=UnitTest_results.trx" --filter "TestCategory!=Category1 & TestCategory!=Category2" --output-format cobertura --output Results/UnitTest.cobertura.xml
`
in which case I get an invalid XML error.  I'm assuming that's due to the fact that `dotnet-coverage` interprets that settings file as its file instead of something to pass to `dotnet test`.

Is there something obvious that I'm missing here?  Is there some way to be more specific about what the `--settings` parameter is referring to or escape it somehow?  I wasn't able to find any relevant examples.  When I run `dotnet test` by itself, everything works as expected both when I provide the arg with `--` and when using `.runsettings`.

## Environment

dotnet-coverage is the latest 17.5.0 release

dotnet --info
.NET SDK (reflecting any global.json):
 Version:   5.0.407
 Commit:    1a1785612e

Runtime Environment:
 OS Name:     Windows
 OS Version:  10.0.19041
 OS Platform: Windows
 RID:         win10-x64
 Base Path:   C:\Program Files\dotnet\sdk\5.0.407\

Host (useful for support):
  Version: 5.0.16
  Commit:  1016676966

.NET SDKs installed:
  5.0.407 [C:\Program Files\dotnet\sdk]

**Comment by jakubch1** [comment]
dotnet-coverage is not supporting runsettings as it is configuration for vstest.console or dotnet test specific.
You should use only `<Configuration>` tag from it. Example here: https://learn.microsoft.com/dotnet/core/additional-tools/dotnet-coverage#settings

Finally, command should be (settings should be passed to `dotnet-coverage` tool, not to `dotnet test`):
`dotnet-coverage collect --settings settings.xml "dotnet test ..."`

**Comment by butterworld** [comment]
Thanks.  I didn't need settings for dotnet-coverage, but rather pass in the settings for dotnet test.  You pointed me in the right direction with your command line example though - the way I got it to work was to put the whole dotnet test command with its arguments inside double quotes and I had to remove spaces around the ampersand character in the filter (msbuild was complaining about it and I couldn't figure out a way to escape it, backtick character in powershell didn't work).
